policyengine_us/parameters/gov/states/mn/tax/income/credits/wfc/phase_in/rate.yaml

```diff
@@ -7,9 +7,9 @@ metadata:
   label: MN WFC phasein rate by number of children
```

Comment:
```suggestion
  label: Minnesota WFC phase in rate by number of children
```

Reply:
Could you make the MN to Minnesota change in all parameter files while you are in here?